.github/workflows/ci.yml

```diff
@@ -128,7 +128,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - run: docker build --target builder .
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
```

Comment:
Btw, thare is `docker/setup-buildx-action@v2`. But I'm not sure if it's compatible

Reply:
Thanks!